climada/test/test_plot.py

```diff
@@ -25,16 +25,26 @@
 import contextily as ctx
 import urllib
 
+from rasterio.windows import Window
 from climada.entity.entity_def import Entity
 from climada.hazard.base import Hazard
 from climada.entity.exposures.base import Exposures
 from climada.entity.impact_funcs.impact_func_set import ImpactFuncSet
 from climada.engine import ImpactCalc, ImpactFreqCurve
-from climada.util.constants import HAZ_DEMO_MAT, ENT_DEMO_TODAY
+from climada.util.constants import HAZ_DEMO_FL, HAZ_DEMO_MAT, ENT_DEMO_TODAY
+
+# Define an exposure to use for plotting
+exp = Exposures.from_raster(HAZ_DEMO_FL, window= Window(10, 20, 50, 60))
 
 class TestPlotter(unittest.TestCase):
     """Test plot functions."""
 
+    def test_plots(self):
+        """ Test plot_scatter, plot_basemap, and plot_raster"""
+        exp.plot_scatter()
+        exp.plot_basemap()
+        exp.plot_raster()
+
```

Comment:
There already is the test `test_exposures_value_pass`. Maybe you could add these lines to that one? It also has an exposure object already defined.

Reply:
Yes good idea, I moved it there, and removed the  exposure that I defined, much more clean now